### PR TITLE
Hide custom mod menu entry icons when adding a11y submenu

### LINKED ISSUE #6551: Modulations custom menu entry buttons stop working after enabling "Add sub-menus for modulation entries"
So, we are at a crossroad. To move further, we have to choose:

1. Hide the icons when that option is enabled (because a11y users don't care about icons anyways - easier route)
2. ~Make those icons work even if the option is enabled (cooler and potentially still useful for sighted users - more work route)~

### PR DESCRIPTION
**Pull request by mkruselj**
Fixes #6551

### CODE REVIEW

File: src/surge-xt/gui/widgets/MenuCustomComponents.cpp [hunk 1]
```diff
@@ -321,15 +347,17 @@ ModMenuCustomComponent::~ModMenuCustomComponent() noexcept = default;
 void ModMenuCustomComponent::getIdealSize(int &idealWidth, int &idealHeight)
 {
     auto menuText = source + " " + amount;
-    getLookAndFeel().getIdealPopupMenuItemSize(menuText, false, 20, idealWidth, idealHeight);
     auto h = idealHeight - 2 * mg;
+
+    getLookAndFeel().getIdealPopupMenuItemSize(menuText, false, 20, idealWidth, idealHeight);
```
Comment (baconpaul) [inline src/surge-xt/gui/widgets/MenuCustomComponents.cpp:352]: Why this change in behavior? I don't understand it.

**Review by baconpaul — COMMENTED** [review]
Just one question otherwise looks great

File: src/surge-xt/gui/widgets/MenuCustomComponents.cpp (hunk 1 above)
Comment (mkruselj) [reply src/surge-xt/gui/widgets/MenuCustomComponents.cpp]: Reverted it!
